### ISSUE TITLE
WGSL sqrt(x) cannot compile

### ISSUE THREAD

**Issue by J-F-Liu** [gfx-rs/naga#418]
Report wgpu error: Validation Error
While pow(x, 0.5) is ok.

**Comment by kvark** [comment]
Could you post the full validation error, please? The code has logic for `sqrt`:
https://github.com/gfx-rs/naga/blob/ac27a1abaaadf142815ae0294e169b1cdb1b743c/src/front/wgsl/conv.rs#L175

**Comment by J-F-Liu** [comment]
Where to see the full validation error, is this?
```
Caused by:
    In Device::create_shader_module
    Failed to parse WGSL
```

**Comment by kvark** [comment]
Actually, we aren't printing the real parsing error here, but it goes into the log. What if you call `env_logger::init()` in your app, do you see the errors from Naga?

**Comment by J-F-Liu** [comment]
See it with env_logger: 
[2021-02-07T07:14:24Z ERROR naga::back::spv::writer] unimplemented math function Sqrt

**Comment by J-F-Liu** [comment]
Well it is because wgpu 0.7 depends on naga 0.3.1, the latest code already fixed this by #401.
